testsuite/domain/src/test/resources/domain-configs/domain-elytron.xml

```diff
@@ -0,0 +1,514 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
If this test case is testing host controller authentication is a complete domain xml really required?